mws/resource_mws_vpc_endpoint_test.go

```diff
@@ -0,0 +1,246 @@
+package mws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/databrickslabs/databricks-terraform/common"
+	"github.com/databrickslabs/databricks-terraform/internal/qa"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMWSVPCEndpoint(t *testing.T) {
```

Comment:
rename to `TestMwsVPCEndpoint`, otherwise automation won't pick it up

Reply:
Done